webapp/src/components/NewsForm.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <form @submit.prevent class="form">
+        <input v-model="newItemTitle" placeholder="News Item Title here">
+        <button @click="addItem">Insert item</button>
```

Comment:
```suggestion
        <button :disabled="!valid" @click="addItem">Insert item</button>
```

What about a computed property `valid` which looks roughly like this:

```js
computed: {
  valid() {
    return this.newItemTitle.length && !this.uniqueTitles.includes(this.newItemTitle)
  }
}
```

Reply:
That's a good idea. In this way we don't have to check if it's already saved. Maybe we can add a hover message for the button, otherwise the user won't know why the button is disabled.

Reply:
Usually you would show a visible validation error on every invalid input field, respectively:

![Screenshot - 2020-10-31T130021 662](https://user-images.githubusercontent.com/2110676/97778680-1313f580-1b79-11eb-9f45-212d9508b1f1.png)


From: https://vuetifyjs.com/en/components/forms/#misc